tests/resources/test_project.py

```diff
@@ -439,12 +708,17 @@ def test_delete_project(collection, session):
     assert expected_call == session.last_call
```

Comment:
Also, according to the ticket description, in v3, users are not supposed to be able to delete projects, anymore. Am I reading that correctly?
https://citrine.atlassian.net/browse/PLA-8395

Reply:
We actually did get that api endpoint, so the ticket is out of date and project delete should work

Reply:
Ah, okay. Can you update or amend the ticket, then, please?

Reply:
done

Reply:
I see it; thanks!